aws_lambda_builders/workflows/python_pip/packager.py

```diff
@@ -131,8 +131,9 @@ def build_dependencies(self, artifacts_dir_path, scratch_dir_path, requirements_
         # by finding/creating a virtualenv of the correct version and when
         # pip is called set the appropriate env vars.
 
+        # Log warning if attempting to build without a manifest.
         if not self.osutils.file_exists(requirements_path):
-            raise RequirementsFileNotFoundError(requirements_path)
```

Comment:
RequirementsFileNotFoundError is no longer referred. You can remove it.

Reply:
- [x] Remove `class RequirementsFileNotFoundError(PackagerError):`